kalite/topic_tools/__init__.py

```diff
@@ -45,12 +44,15 @@
 # Globals that can be filled
 TOPICS          = None
 CACHE_VARS.append("TOPICS")
-def get_topic_tree(force=False, annotate=False, channel=settings.CHANNEL):
+def get_topic_tree(force=False, annotate=False, channel=settings.CHANNEL, language="en"):
+    translation.activate(language)
     global TOPICS, TOPICS_FILEPATHS
     if not TOPICS:
         TOPICS = {}
     if TOPICS.get(channel) is None:
-        TOPICS[channel] = softload_json(TOPICS_FILEPATHS.get(channel), logger=logging.debug, raises=False)
+        TOPICS[channel] = {}
+    if TOPICS.get(channel, {}).get(language) is None:
```

Comment:
Otherwise the approach seems good, and isn't too far from what we eventually want.